src/Pact/Analyze/Term.hs

```diff
@@ -36,19 +45,24 @@ etermEType (ETerm _ ety)   = EType ety
 etermEType (EObject _ sch) = EObjectTy sch
 
 data Term ret where
```

Comment:
This is the other style of term, where we have `PureTerm` and all the other constructs are at the same level.